src/simpledb/opt/TablePlanner.java

```diff
@@ -91,7 +96,7 @@ private Plan makeIndexSelect() {
          Constant val = mypred.equatesWithConstant(fldname);
          if (val != null) {
             IndexInfo ii = indexes.get(fldname);
-            System.out.println("index on " + fldname + " used");
+            System.out.printf("Running Index Scan using " + fldname + "(cost=" + ii.blocksAccessed() + ")\n");

```

Comment:
maybe can add the index type (hash or btree)

Reply:
good idea